src/config-import.js

```diff
@@ -177,6 +172,17 @@ export async function readConfig(configPath) {
         return sort - nextSort
       })
 
+      // check that preset references existing icon
+      const iconFilenames = new Set(
+        iconEntries.map((icon) => {
+          const matches = path.basename(icon.filename).match(ICON_NAME_REGEX)
+          if (matches) {
+            const [_, name] = matches
+            return name
+          }
+        })
+      )
```

Comment:
Because the `map` function can return `undefined`, this this variable is `Set<undefined | string>` instead of what you probably intended, which is `Set<string>`. I opened #790 to fix that.